### ISSUE TITLE
Cannot connect to SmartLife

### ISSUE THREAD

**Issue by thibp** [sabinus52/jeedom-smartlife#59]
I configure Jeedom plugin with Username / Password (e-mail / password), then I leave phone empty, then select SmartLife, then Test connection.
I get this error on Debug :

> [2021-02-01 00:09:41][DEBUG] : CHECK CONNECTION : Start
> [2021-02-01 00:09:41][DEBUG] : CONNECTION : *masked*@*masked*.*masked* () smart_life - 5 s
> [2021-02-01 00:09:41][ERROR] : Erreur de connexion au cloud Tuya : Get accesstoken failed. Invalid parms.
> [2021-02-01 00:09:41][DEBUG] : CHECK CONNECTION : Exception Object (     [message:protected] => Get accesstoken failed. Invalid parms.     [string:Exception:private] =>      [code:protected] => 0     [file:protected] => /var/www/html/plugins/SmartLife/vendor/sabinus52/tuyacloudapi/src/Session/Session.php     [line:protected] => 225     [trace:Exception:private] => Array         (             [0] => Array                 (                     [file] => /var/www/html/plugins/SmartLife/vendor/sabinus52/tuyacloudapi/src/Session/Session.php                     [line] => 177                     [function] => checkResponse                     [class] => Sabinus\TuyaCloudApi\Session\Session                     [type] => ->                     [args] => Array                         (                             [0] => Array                                 (                                     [responseStatus] => error                                     [errorMsg] => Get accesstoken failed. Invalid parms.                                 )                              [1] => Get accesstoken failed. Invalid parms.                         )                  )              [1] => Array                 (                     [file] => /var/www/html/plugins/SmartLife/vendor/sabinus52/tuyacloudapi/src/Session/Session.php                     [line] => 118                     [function] => _createToken                     [class] => Sabinus\TuyaCloudApi\Session\Session                     [type] => ->                     [args] => Array                         (                         )                  )              [2] => Array                 (                     [file] => /var/www/html/plugins/SmartLife/vendor/sabinus52/tuyacloudapi/src/TuyaCloudApi.php                     [line] => 119                     [function] => getToken                     [class] => Sabinus\TuyaCloudApi\Session\Session                     [type] => ->                     [args] => Array                         (                         )                  )              [3] => Array                 (                     [file] => /var/www/html/plugins/SmartLife/vendor/sabinus52/tuyacloudapi/src/TuyaCloudApi.php                     [line] => 52                     [function] => _request                     [class] => Sabinus\TuyaCloudApi\TuyaCloudApi                     [type] => ->                     [args] => Array                         (                             [0] => Discovery                             [1] => discovery                         )                  )              [4] => Array                 (                     [file] => /var/www/html/plugins/SmartLife/core/class/SmartLife.class.php                     [line] => 68                     [function] => discoverDevices                     [class] => Sabinus\TuyaCloudApi\TuyaCloudApi                     [type] => ->                     [args] => Array                         (                         )                  )              [5] => Array                 (                     [file] => /var/www/html/plugins/SmartLife/core/ajax/SmartLife.ajax.php                     [line] => 34                     [function] => checkConnection                     [class] => SmartLife                     [type] => ::                     [args] => Array                         (                         )                  )          )      [previous:Exception:private] =>  )
> [2021-02-01 00:09:41][DEBUG] : CHECK CONNECTION : End

It seems like, it connects only to Tuya not to SmartLife, because with curl command I get :

> { [146 bytes data]
> 100   239  100   146  100    93    111     70  0:00:01  0:00:01 --:--:--   182HTTP/2 200
> date: Sun, 31 Jan 2021 23:02:21 GMT
> content-type: application/json;charset=ISO-8859-1
> content-length: 146
> server: Apache-Coyote/1.1
> 
> {"access_token":"***********","refresh_token":"*************","token_type":"bearer","expires_in":864000}
> * Connection #0 to host px1.tuyaus.com left intact

**Comment by thibp** [comment]
It is now working, Country has to be set to 33
